### ISSUE TITLE
How can I use float in NumericInput

### ISSUE THREAD

**Issue by KirillBychkov** [palantir/blueprint#3949]
Hello everyone. Who can tell me please, what I doing wrong? Why its convert my float to int? How can I use float in NumericInput?

```
import React, { useState } from 'react';
import { NumericInput } from '@blueprintjs/core';

export default () => {
  const [state, setState] = useState(0.5);

  return (
      <NumericInput
        value={state}
        stepSize={0.5}
        max={6}
        min={0}
      />
  );
};
```
![Screenshot_20200130_094623](https://user-images.githubusercontent.com/32789602/73429908-6be20380-4345-11ea-86da-4050b862ad73.png)

    "@blueprintjs/core": "^3.22.3",
    "react": "^16.12.0",

**Comment by KirillBychkov** [comment]
Okey, if I comment out  // max={6} and //  min={0} it works fine. But issue is still actual. How can I use it with min and max props?

**Comment by adidahiya** [comment]
hi @KirillBychkov, please fill out the bug report template completely, it helps debug your issue. which version of blueprint are you using? you may be encountering this bug which was fixed in a recent version: https://github.com/palantir/blueprint/issues/3889

**Comment by KirillBychkov** [comment]
"@blueprintjs/core": "^3.22.3",
"react": "^16.12.0",

**Comment by adidahiya** [comment]
@KirillBychkov the issue I linked above was fixed in 3.23. please try again with the latest version. if you still have the issue then please create a code sandbox to show the issue (as shown in the issue template)

**Comment by KirillBychkov** [comment]
Thanks)
